dev/com.ibm.ws.org.apache.cxf.cxf.rt.rs.service.description.3.2/src/org/apache/cxf/jaxrs/model/wadl/WadlGenerator.java

```diff
@@ -219,9 +219,7 @@ public void filter(ContainerRequestContext context) {
     }
 
     protected void doFilter(ContainerRequestContext context, Message m) {
-        //Liberty code change start
-        if (!"GET".equals(((MessageImpl) m).getHttpRequestMethod())) {
-            //Liberty code change end
+        if (!"GET".equals(m.get(Message.HTTP_REQUEST_METHOD))) {
```

Comment:
We can delete this overlay file - the only other changes to it have been made in CXF (and only really apply if we have the file in our source tree anyway).